src/main/test/biscuit/parser/ParserTest.java

```diff
@@ -0,0 +1,67 @@
+package biscuit.parser;
+
+
+import biscuit.commands.Command;
+import biscuit.exceptions.BiscuitException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Unit test for Parser class.
+ */
+class ParserTest {
```

Comment:
Very detailed list of testing provided. Great job! 🙂

Reply:
Thank you!